src/installer/installer.go

```diff
@@ -324,7 +318,12 @@ func (i *installer) getInventoryHostsMap(hostsMap map[string]inventory_client.Ho
 			return nil, err
 		}
 		// no need for current host
-		delete(hostsMap, i.Hostname)
```

Comment:
This was added after my initial change to remove the `Hostname` field.

Generally I feel like this map should be keyed by id, but I don't think that kind of change is in the scope of this PR. Additionally I would rather not have to provide the hostname to the installer just to save a few iterations through this map.